src/main/java/picard/vcf/CollectVariantCallingMetrics.java

```diff
@@ -293,8 +312,16 @@ public void calculateDerivedFields() {
             super.calculateDerivedFields();
             // Divide by zero should be OK -- NaN should get propagated to metrics file.
```

Comment:
ouch. NaNs will bite. need to have a solution for NaNs. can you add a test to see what will happen and protect against it?